src/function_patcher.cpp

```diff
@@ -158,21 +158,21 @@ void DoAudioMagic(int16_t *addr, uint32_t size, bool isDRC, AIInitDMAfn targetFu
             } else if (gCurAudioMode == AUDIO_MODE_LEFT_TV_RIGHT_DRC) {
                 for (uint32_t i = 0; i < 0x120; i += 2) {
                     // Mix down TV to MONO and put it in the left channel
-                    auto val = (((int32_t) TVCopy[i] + (int32_t) TVCopy[i + 1]) >> 1);
+                    auto val = (((int32_t) sTVCopy[i] + (int32_t) sTVCopy[i + 1]) >> 1);
                     if (val > 0x7FFF) {
                         val = 0x7FFF;
                     } else if (val < -0x8000) {
                         val = 0x8000;
                     }
-                    addr[i] = (int16_t) val;
+                    addr[i + 1] = (int16_t) val;
                     // Mix down DRC to MONO and put it in the right channel
-                    val = (((int32_t) DRCCopy[i] + (int32_t) DRCCopy[i + 1]) >> 1);
+                    val = (((int32_t) sDRCCopy[i] + (int32_t) sDRCCopy[i + 1]) >> 1);
                     if (val > 0x7FFF) {
                         val = 0x7FFF;
                     } else if (val < -0x8000) {
                         val = 0x8000;
                     }
-                    addr[i + 1] = (int16_t) val;
+                    addr[i] = (int16_t) val;
```

Comment:
I'm not sure about this change, aren't the audio stream samples stored Left then Right? The combined mode seems to follow L/R order and sounded correct per my testing.